pkgs/development/libraries/libgssglue/default.nix

```
@@ -1,13 +1,36 @@
-{ lib, stdenv, fetchurl, libkrb5 }:
+{
+  lib,
+  stdenv,
+  fetchFromGitLab,
+  autoconf,
+  automake,
+  libtool,
+  libkrb5,
+}:
 
-stdenv.mkDerivation rec {
+stdenv.mkDerivation (finalAttrs: {
   pname = "libgssglue";
-  version = "0.4";
+  version = "0.9.1";
 
-  src = fetchurl {
-    url = "http://www.citi.umich.edu/projects/nfsv4/linux/libgssglue/${pname}-${version}.tar.gz";
-    sha256 = "0fh475kxzlabwz30wz3bf7i8kfqiqzhfahayx3jj79rba1sily9z";
-  };
+  src =
+    let
+      tag = with lib.versions; "${majorMinor finalAttrs.version}-${patch finalAttrs.version}";
+    in
+    fetchFromGitLab {
+      domain = "salsa.debian.org";
+      owner = "debian";
+      repo = "libgssglue";
+      rev = "tags/debian/${tag}";
+      hash = "sha256-D+Us4wk0Fh4Idco8HKcF1EStcYyWWoUuXRshAQAgJtk=";
+    };
```

Please use upstream source and not the debian mirror.
https://gitlab.com/gsasl/libgssglue

```suggestion
  src = fetchFromGitLab {
    owner = "gsasl";
    repo = "libgssglue";
    rev = "libgssglue${finalAttrs.version}";
    hash = "";
  };
```